src/constants/pathSegment.ts

```diff
@@ -0,0 +1,3 @@
+export const PathSegment = {
```

Comment:
константы принято именовать либо в camelCase, либо в UPPER_CASE
```suggestion
export const pathSegments = {
```

Reply:
готово